frontend/ui/src/widgets/property-sheet/color-picker.tsx

```
@@ -0,0 +1,204 @@
+            type="button"
+            className="h-6 w-6 cursor-pointer rounded-sm transition-transform hover:scale-110"
+            style={{ backgroundColor: `${color}` }}
+            onClick={() => onColorSelect({ hex: color })}
+            title={color}
+          />
+        ))}
+      </div>
+
+      <div className="flex justify-between">
+        {colorGroups.map((group, groupIndex) => (
+          <div key={`group-${groupIndex}`} className="flex flex-col gap-1">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)